config/initializers/action_mailer.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.configure do
+  config.action_mailer.delivery_method = :smtp
+
+  config.action_mailer.smtp_settings = {
+    user_name: Rails.application.secrets.mailer_username,
+    password: Rails.application.secrets.mailer_password,
+    address: Rails.application.secrets.mailer_address,
+    domain: Rails.application.secrets.mailer_domain,
+    port: '2525',
```

Comment:
Thought this is ok, we could move it to an ENV variable if in the future we need to change some of the configuration